.gitignore

```diff
@@ -45,3 +45,6 @@ rholang/src/main/java/coop/**
 ###################
 !rosette/bin/README
 rosette/src/*.o
+rosette/.vagrant/
```

Comment:
Unrelated change

Reply:
Do you want me to open another PR for this? I added this as a part of this PR because I didn't want to accidentaly commit my Vagrant related files.

Reply:
Yes, please remove it.

Reply:
Removed.